kolibri_dynamic_collections_plugin/assets/src/views/CollectionEditorPage.vue

```diff
@@ -161,6 +169,17 @@
     methods: {
       ...mapActions(['resetCollectionEditorState']),
       ...mapActions('collectionBase', ['addChannels', 'removeChannel', 'setMetadata']),
+      exportAsJSON() {
+        // TODO: Instead of creating a blob here, add an API endpoint which
+        //       returns a JSON file and window.open() that.
```

Comment:
What would be the benefit of having an API endpoint? I see that this code is moved in upcoming commits to the actions `coreCollections/actions.js`, that's great.